test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Do you need this file for testing ? If not remove this.

Reply:
Removed this test file